org.civicrm.styleguide/CRM/Styleguide/Manager.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+class CRM_Styleguide_Manager {
+
+  /**
+   * @var array
+   */
+  private $all = NULL;
+
+  /**
+   * Get the definition of specific style-guide by name.
```

Comment:
Fair enough. 

When the phrase appears by itself (like in a page-title), it doesn't matter much. For prose in a sentence, I find "style guide" harder to read than "style-guide". The issue is that both words are multipurpose (e.g. "style" can be noun or verb; "guide" can be noun or verb), and you have to absorb more of the sentence to be sure of the meaning. With the "-", you can rest assured that it's intended as the compound noun.